app/assets/javascripts/map.js

```diff
@@ -0,0 +1,189 @@
+var locations = [

```

Comment:
seems like a lot of static data in this file.  Maybe we need an object here to wrap this data then feed the list with an api call.